src/components/Counter.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class Counter extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = { value: 0 }
+      this.increase = this.increase.bind(this);
+      this.decrease = this.decrease.bind(this);
+    }
+
+    componentDidMount() {
+      const root = ReactDOM.createRoot(
```

Comment:
Here you create completely new React app and render your components in new React Dom instance. You meant to use [React.createElement](https://reactjs.org/docs/react-api.html#createelement)? In this case please use it.
You need to `return React.createElement(...)`

Reply:
Please do it in render function.
componentDidMount's original purpose is to set state only. Dom measurements may be, but not render

Reply:
Fixed.